src/coreclr/jit/codegenlinear.cpp

```diff
@@ -1657,7 +1657,7 @@ void CodeGen::genConsumeRegs(GenTree* tree)
 #ifdef FEATURE_SIMD
             // (In)Equality operation that produces bool result, when compared
             // against Vector zero, marks its Vector Zero operand as contained.
-            assert(tree->OperIsLeaf() || tree->IsSIMDZero());
+            assert(tree->OperIsLeaf() || tree->IsSIMDZero() || tree->IsVectorZero());
```

Comment:
Just noting that we're getting close to the point where I can remove `IsSIMDZero` as basically everything has moved forward to use HWIntrinsics under the covers.

Reply:
Good to know.